src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/Symbolic/BitVectorSolver.cs

```diff
@@ -28,73 +28,73 @@ public BitVectorAlgebra(BDD[] minterms)
             }
             _mintermVectors = singleBitVectors;
 
-            False = BitVector.CreateFalse(minterms.Length);
-            True = BitVector.CreateTrue(minterms.Length);
+            Empty = BitVector.CreateFalse(minterms.Length);
+            Full = BitVector.CreateTrue(minterms.Length);
         }
 
-        public BitVector False { get; }
-        public BitVector True { get; }
+        public BitVector Empty { get; }
+        public BitVector Full { get; }
 
-        public bool AreEquivalent(BitVector predicate1, BitVector predicate2) => predicate1.Equals(predicate2);
+        public bool IsFull(BitVector set) => set.Equals(Full);
```

Comment:
NIT: this doesn't seem to be covered in our tests

Reply:
There's a non-trivial amount of code not covered by our tests, unfortunately.  I'd asked @olsaarik a few weeks back if he could take a pass at adding patterns/inputs that would cover more things, and I think he was planning to do so after he's done with the more pressing fixes we've discussed. Olli, is that still something you're planning to do?

Reply:
That had fallen off my mind, but yes, that is something I should do. I can take it up after the matching logic changes.

Reply:
Thanks, Olli.  I'll open an issue to track that.